guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc

```diff
@@ -59,6 +59,26 @@ fdi.ipv6.method::
 This option overrides it, set it to `ignore` to disable the IPv6 stack.
 This option only works in DHCP mode.
 
+fdi.zips::
+  Filenames with extensions to be downloaded and started during boot.
+For more information, see see xref:Extending_the_Discovery_Image[].
+
+fdi.zipserver::
+  TFTP server to use to download extensions from.
+For more information, see see xref:Extending_the_Discovery_Image[].
+
+fdi.countdown::
+  Number of seconds to wait until text-user interface is refreshed after the initial discovery attempt.
+This value defaults to 45 second.
+Increase this value when status page reports N/A.
```

Comment:
```suggestion
Increase this value when status page reports N/A.
```

Is N/A the exact thing that is reported? Or is there some error, or just a hanging black void that stares back at the user as the user stares into it?

Reply:
We get this reported quite often, TUI reads status of the background job and when it's not yet done (slow network) it prints IP Address: N/A.

Reply:
Cool, fine by me if that is what the user will see.